src/js/nizovi.js

```diff
@@ -0,0 +1,129 @@
+/* eslint-disable no-unused-vars */
+// 1. zadatak
+const slova = ['s', 'a', 'e']
+// 2. zadatak
+const brojevi = [1, 2, 3]
+
+// 4. zadatak
+const niz = []
+const max = 10000000
+const min = 1000000
+
+for (let i = 0; i < 50; i++) {
+  niz[i] = Math.floor(Math.random() * (max - min + 1)) + min
+}
+// eslint-disable-next-line no-console
+console.log(niz)
+
+// 5. zadatak
+const alphabet = 'abcdefghijklmnopqrstuvwxyz'.split('')
+const niz5 = []
+
+for (let i = 0; i < 40; i++) {
+  niz5[i] = alphabet[Math.floor(Math.random() * 25)]
+}
+// console.log(niz5)
+
+// 6. zadatak
+const niz2 = [['a', 'b', 'c'], ['d', 'e', 'f'], ['g', 'h', 'i'], ['q', 't', 'u']]
+
+// 9. zadatak
+let el = ''
+for (el of alphabet) { // radi mi samo for...of,a ne for...in
+  // console.log(el)
+}
+// 10. zadatak
+const srpSlova = 'абвгдђжзијклљмнњопрстћуфхцчџш'.split('')
+
+// 11. zadatak     Nisam siguran da li se na ovo misli u ovom zadatku
```

Comment:
Ne već ako je pozovemo sa:

```cirilica('Luka')```
Ona vrati
```Лука```

Reply:
Ahaaaaam. Okej, pogledacu sada to.